.github/workflows/ci-additional.yml

```diff
@@ -5,7 +5,6 @@ on: [push, pull_request]
 jobs:
   mindeps:
     runs-on: "ubuntu-latest"
-    if: "contains(github.event.head_commit.message, 'test-mindeps') || github.event_name != 'pull_request'"
```

Comment:
Could you add a TODO comment that we should figure out how to make this work?

Reply:
I don't think there's any reason for the limitation. It will always be run in paralell with the other workflows (I just tested you can start up to 60 in parallel!) and it will always be faster.

Reply:
Do you know if that's 60 per repository, or 60 per GitHub org?

Reply:
https://docs.github.com/en/actions/reference/usage-limits-billing-and-administration#usage-limits

Afraid it doesn't answer your specific question though. We could test it by kicking off a torture test in two different repos I guess.

@shoyer the above link also seems to answer your question from a few days back: we are currently capped at 5 MacOS workflows at any given time; it looks like upgrading to the "enterprise" plan would raise that limit to 50.

Reply:
Re-reading it, it says "in your account"

Reply:
I interpret that as meaning 60 concurrent builds for repos in the Dask GitHub org. Perhaps for now we should continue to only run these builds on `master` to not hog shared CI resources. We can always reevaluate later.

Reply:
Shouldn't we instead start disabling less-than-essential builds only after we start experiencing wait time? So far, it has never happened on Linux/Windows as far as I could see.
The thing is: if this workflow had been enabled at PR time, I would have never introduced the regression in the test suite to begin with. The build in master is rarely something developers will remember to check.

Reply:
Also, if you're worried about running out of slots, I'd much rather remove the on: push (which nobody looks at anyway) and replace it with a nightly build. Such a change would help a lot with the Mac slots too.

Reply:
Sure, we can give this a try to see what happens. Do you know if the `on: push` builds apply to the Dask org account or the personal accounts for the user is who submitting a PR from (i.e. from their fork of `dask`)?

For reference, I think at one point @jacobtomlinson mentioned finding the `on: push` builds useful (though I don't recall where that comment was)

Reply:
> Do you know if the on: push builds apply to the Dask org account or the personal accounts for the user is who submitting a PR from (i.e. from their fork of dask)?

No evidence, but I would be inclined to believe the former as it's the one that makes by far more sense commercially